src/models/treemodel.cpp

```diff
@@ -74,7 +74,7 @@ QVariant BottomUpModel::rowData(const Data::BottomUp* row, int column, int role)
     if (role == Qt::DisplayRole || role == SortRole) {
         switch (column) {
         case Symbol:
-            return row->symbol.symbol.isEmpty() ? tr("??") : row->symbol.prettySymbol;
+            return row->symbol.getNonEmptySymbol();
```

Comment:
Util::formatSymbol(row->symbol)

Reply:
This and all the other comments should now be fixed. Please have a look.